app/javascript/packs/articlePage.jsx

```diff
@@ -88,26 +87,13 @@ function showAnnouncer() {
   document.getElementById('article-copy-link-announcer').hidden = false;
 }
 
-// Temporary Ahoy Stats for displaying comments section either on page load or after scrolling
-function trackCommentsSectionDisplayed() {
-  const callback = (entries, observer) => {
-    entries.forEach((entry) => {
-      if (entry.isIntersecting) {
-        ahoy.track('Comment section viewable', { page: location.href });
-        observer.disconnect();
-      }
-      if (location.hash === '#comments') {
-        //handle focus event on text area
-        const element = document.getElementById('text-area');
-        const event = new FocusEvent('focus');
```

Comment:
Ah, I see it now :+1:

Reply:
The method was doing 2 things (tracking the event + focusing), so it was harder to spot.